src/components/Card/index.tsx

```diff
@@ -0,0 +1,40 @@
+import Header from './Header'
+import CardImage from './CardImage'
+import HpDmgFooter from './HpDmgFooter'
+import Abilities from './Abilities'
+import Description from './Description'
+import PlayCost from './PlayCost'
+import Type from './Type'
+import './card.css'
+
+interface CardProps {
+    rarity: string;
+    name: string;
+    playCost: number;
+    imagePath: string;
+    abilities: number[];
+    type: string;
+    description: string;
+    hp: number;
+    damage: number;
+  }
+  
+  const Card: React.FC<CardProps> = ({ rarity, name, playCost, imagePath, abilities, type, description, hp, damage }) => {
```

Comment:
I think the wrapper is going to provide a single object that would have all these fields. Doesn't need to be changed now, just something to keep in mind.